### DISCUSSION TITLE
How to I print a RUST_BACKTRACE=1-style backtrace of an ERROR entry printed by tracing-subscriber?

### DISCUSSION THREAD

**Discussion by vi** [tokio-rs/tracing#2841] [Q&A]
I see just a single line: `<ts> ERROR <module> <message>` from the code that uses simple `tracing_subscriber::fmt::init()`.

How do I make it also capture and print the call stack?
Is there some panic-on-error environment variable or something like that?

**Comment by clarkmcc** [comment]
Bumping, I'd also like to know

**Comment by kaffarell** [comment]
AFAIK this is not built into tracing right now. You can print a backtrace using https://doc.rust-lang.org/std/backtrace/index.html though.
